data/migrations/20191002212356_farmers.js

```diff
@@ -1,5 +1,5 @@
 exports.up = function(knex) {
-  return knex.schema.createTable("farmers", f => {
+  return knex.schema.createTable("farms", f => {
```

Comment:
:question: 
Why change it to farms?
Is this table necessary at all anymore?

Reply:
For clarity, and so a single user id can have multiple farms

Reply:
Ahh, it can by the nature of having separate locationID's but a single userID